manifests/c/CONZEInformatik/Vistrax/3.3.23061.1234/CONZEInformatik.Vistrax.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.2.10 $debug=NVS0.CRLF.5-1-22621-2428.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: CONZEInformatik.Vistrax
+PackageVersion: 3.3.23061.1234
+InstallerLocale: en-US
+InstallerType: zip
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
+UpgradeBehavior: install
+ReleaseDate: 2023-03-02
```

Comment:
```suggestion
ReleaseDate: 2023-03-02
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.Framework.DeveloperPack_4
```

Reply:
Thank you!
The 64bit one dependency should be fine from what I have read on the manufacturer website.